bin/dev

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+if gem list --no-installed --exact --silent foreman; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+# Default to port 3000 if not specified
+export PORT="${PORT:-3001}"
+
+exec foreman start -f Procfile "$@"
```

Comment:
If there's already a default specified in `Procfile`, does it need to be re-specified here?

I'd wonder if we need this file at all, or if the original Makefile target code would suffice (`foreman start`)

Reply:
updated in https://github.com/18F/identity-dashboard/commit/594bc2dafc173305e4c2b377dcf9ad97baabc8e9